src/Async.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * The `Async` type reperesents a "lazy" or "cold" asynchronous
```

Comment:
lol nice catch :back-away:

Reply:
Yay for the spell checker VS code extension :joy: